static/searchPage.html

```diff
@@ -3,59 +3,52 @@
   <head>
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
-    <script src="https://maps.googleapis.com/maps/api/js?key=APIkey"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAM2CwINrDFcPP4uoEKZ1iVlPgzUryWgfc"></script>
```

Comment:
Are we adding the API key here because it is restricted to certain addresses anyway? On an earlier PR, someone mentioned not committing the api key here, which is why it was just "APIkey".

Reply:
Oh whoops! My bad. I guess it was inevitable this happened because we have to replace the "APIkey" with an actual API key when running/testing the server. I'll see what to do about this.